tasks/install.yml

```diff
@@ -22,7 +22,7 @@
   template:
     src: ".pyenvrc.j2"
     dest: "{{ pyenv_path }}/.pyenvrc"
-    owner: "{{ pyenv_owner }}"
```

Comment:
I understood instead of replacing `owner` you wanted to add the attribute `group`, right? https://docs.ansible.com/ansible/latest/collections/ansible/builtin/template_module.html#parameter-group

Reply:
Arh, sorry, my bad, I didn't modify the correct file, this is the file I wanted to modify : https://github.com/staticdev/ansible-role-pyenv/pull/128/files#diff-98633e3bbe17d6f9687d8d15ebb7af031e8b7d85a99421e2d70a7f1cc6786faf

Reply:
@ni-fgenois are you able to verify with this code that your issue on Mac is solved?

Reply:
Yep! Works for me, just re-tested everything